.github/dependabot.yml

```diff
@@ -4,6 +4,13 @@ updates:
     directory: /
     schedule:
       interval: daily
+    groups:
+      prost:
+        patterns:
+          - "prost*"
+      tonic:
+        patterns:
+          - "tonic*"
```

Comment:
Don't these dependencies have to be updated _together_?

Reply:
good question. i noticed these are not maintained as part of the same organization _(see repository url's below)_, and we don't update these together in the [proxy](https://github.com/linkerd/linkerd2-proxy/blob/main/.github/dependabot.yml#L37-L47) at the moment.

i think you're right though. `prost` generates code based on the `tonic` interfaces, so.. we should probably do the same thing as the [control plane](https://github.com/linkerd/linkerd2/blob/main/.github/dependabot.yml#L81-L86).

- https://github.com/tokio-rs/prost/
- https://github.com/hyperium/tonic

Reply:
linkerd/linkerd2-proxy#3821 consolidates these groups together. i can do the same here, i'll mark this as a draft in the meantime.